css/style.css

```diff
@@ -529,6 +748,23 @@ header{
     /* line-height: 40px; */
 }
 
+.location-in-focus__text-element__header::after{
+    position: relative;
+    display: block;
+    content: '';
+    height: 3px;
+    width: 0%;
+    background-color: var(--color-light-blue);
+    margin-top: 7px;
+    transition: var(--animation-speed) ease-in-out;
+}
+
+.location-in-focus__text-element__header:hover::after
+{
```

Comment:
Looks like these styles are copy-paste, could you pls move them to some reusable class, and reuse it